src/test/util_threadnames_tests.cpp

```diff
@@ -26,15 +26,17 @@ const std::string TEST_THREAD_NAME_BASE = "test_thread.";
  *
  * @return the set of name each thread has after attempted renaming.
  */
+
+static Mutex lock;
+static std::set<std::string> names GUARDED_BY(lock);
+
 std::set<std::string> RenameEnMasse(int num_threads)
 {
     std::vector<std::thread> threads;
-    std::set<std::string> names;
-    std::mutex lock;
 
     auto RenameThisThread = [&](int i) {
         util::ThreadRename(TEST_THREAD_NAME_BASE + ToString(i));
-        std::lock_guard<std::mutex> guard(lock);
+        LOCK(lock);
         names.insert(util::ThreadGetInternalName());
     };
```

Comment:
bbe87c91a8594e33542bae3d738499324cd06906

It seems nothing to capture now:
```suggestion
    auto RenameThisThread = [](int i) {
        util::ThreadRename(TEST_THREAD_NAME_BASE + ToString(i));
        std::lock_guard<std::mutex> guard(lock);
        LOCK(lock);
        names.insert(util::ThreadGetInternalName());
    };
```

Reply:
Dropped redundant capture